### ISSUE TITLE
Adding: Wouter Michiel

### ISSUE THREAD

**Issue by LucaMarconato** [scverse/governance#67]
# Onboarding checklist

Welcome @melonora!

- [ ] add to website
- [ ] add to core@scverse.org google group. Note: the email address used here will also get access to the shared google drive and calendars (see [here](https://docs.google.com/document/d/1VbmcPM5pivthVnDHH0UgX7zITQFIJyahDjI_WYu7xzE/edit#heading=h.rjvd7yl35x9c))
- [ ] add to zulip
    - [ ] @core team
    - [ ] core-team private channel (see [here](https://docs.google.com/document/d/1VbmcPM5pivthVnDHH0UgX7zITQFIJyahDjI_WYu7xzE/edit#heading=h.6idmmugj38ay))
- [ ] Grant "Leader" trust level on discourse
- [ ] add to password manager
- [ ] add to github org and [scverse/core](https://github.com/orgs/scverse/teams/core) github team
- [ ] Add to NumFOCUS slack
- [ ] Send them link to [core team handbook](https://docs.google.com/document/d/1VbmcPM5pivthVnDHH0UgX7zITQFIJyahDjI_WYu7xzE/edit?usp=sharing)
- [ ] announce the new member on twitter

**Comment by LucaMarconato** [comment]
I don't have the permissions to add Wouter to the scverse GitHub organization, could you help please @flying-sheep?

**Comment by melonora** [comment]
Numfocus can be checked already. Was already there in my role as napari core dev.

**Comment by flying-sheep** [comment]
@LucaMarconato someone already did!

**Comment by LucaMarconato** [comment]
True @flying-sheep, he is now in the scverse org, thanks for checking. I noticed that he is still not in this list https://github.com/orgs/scverse/teams/core, can you help with that please?

**Comment by LucaMarconato** [comment]
@ivirshup I tried and I think I don't have the permissions to grant the "leader" trust level to Wouter in the discourse. Could you please double check: https://discourse.scverse.org/u/melonora/summary

**Comment by grst** [comment]
wouter doesn't appear to have an scverse.org email. @mikelkou could you please add him?

**Comment by mikelkou** [comment]
Of course. I need him (@melonora ) to give me a personal email as a backup.

**Comment by melonora** [comment]
done

**Comment by mikelkou** [comment]
email done! michiel.vierdag@scverse.org
